sources/packages/web/src/views/aest/NoticeOfAssessment.vue

```diff
@@ -40,7 +44,25 @@ export default defineComponent({
     },
   },
   setup() {
-    return { AESTRoutesConst };
+    const snackBar = useSnackBar();
+    const reissueMSFAA = async (
+      applicationId: number,
+      reloadNOA: () => Promise<void>,
```

Comment:
For my understanding: does `Promise<void>` make the component to reload?

Reply:
it means that we can pass the function a parameter and we can trigger the passed function parameter inside the function

Reply:
Sorry, after taking another look at it I got it. Thanks.